LayoutTests/fast/scrolling/mac/scrollbars/scrollbar-width-dynamic-none-to-auto.html

```diff
@@ -0,0 +1,30 @@
+<!-- webkit-test-runner [ MockScrollbarsEnabled=false AsyncOverflowScrollingEnabled=true ] -->
+<!doctype html>
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<style>
+    .container {
+        background-color: orange;
+        overflow: auto;
+        width: 200px;
+        height: 200px;
+        border: 1px solid black;
+    }
+
+    .content {
+        background-color: green;
+        height: 300px;
+        width: 100%;
+    }
+</style>
+    <div id="box" class="container" style="scrollbar-width: none;">
+        <div class="content"></div>
+    </div>
+<script>
+    if (window.testRunner)
+        testRunner.waitUntilDone();
+    internals.setUsesOverlayScrollbars(false);
+    requestAnimationFrame(() => requestAnimationFrame(() => {
+        document.getElementById('box').style.scrollbarWidth = 'auto';
```

Comment:
The way I like to do this is `document.body.classList.add('changed')` and write a `body.changed .box` selector with the new style.

Reply:
Fixed.